#!/bin/sh /usr/share/dpatch/dpatch-run
## 906-file-hurd.dpatch by Michael Casadevall <sonicmctails@gmail.com>
##
## DP: Fixes a FTBFS on hurd (Closes: #433716).

@DPATCH@

diff -Naur file-4.21.orig/src/magic.c file-4.21/src/magic.c
--- file-4.21.orig/src/magic.c	2007-03-26 17:59:50.000000000 +0000
+++ file-4.21/src/magic.c	2007-07-19 06:25:16.000000000 +0000
@@ -58,6 +58,11 @@
 #include <locale.h>
 #endif
 
+#ifndef PIPE_BUF
+// Get the PIPE_BUF from pathconf
+#define PIPE_BUF pathconf(".", _PC_PIPE_BUF)
+#endif
+
 #include <netinet/in.h>		/* for byte swapping */
 
 #include "patchlevel.h"
